src/ctelnet.cpp

```diff
@@ -1390,18 +1390,10 @@ void cTelnet::postMessage(QString msg)
 
 //forward data for further processing
 
-
-void cTelnet::gotPrompt(string& mud_data)
+// Patch for servers that need GA/EOR for prompt fixups
+void cTelnet::applyGAFix()
 {
-    mpPostingTimer->stop();
-    mMudData += mud_data;
-
     if (mUSE_IRE_DRIVER_BUGFIX && mGA_Driver) {
```

Comment:
Given that the fix had been move to this method, shouldn't this test be what determines that `applyGAFix()` gets called in the first place (put the test outside of the method)...